### ISSUE TITLE
Support for Type AAAA Entries (IPv6)

### ISSUE THREAD

**Issue by Sajito** [mageddo/dns-proxy-server#208]
### What is expected

It would be nice if we could specify type AAAA entries, to make the dns server resolve hostnames to ipv6 addresses. 

### What is Happening

Right now there is no way to add any type AAAA entries, making it impossible to resolve a hostname to the ipv6 address.

**Comment by 4ntsu** [comment]
I would love to see this functionality too. My Docker network is dual stack, and it would be great if the DNS server could return both the v4 and v6 addresses when a container is queried.

**Comment by mageddo** [comment]
Yeah it would be a nice feature.

I'm working to increase Mac and windows compatibility right now, this one could be done in sequence

**Comment by mageddo** [comment]
Hey guys, I'm releasing #390 (version 3.15) snapshot  right now, it will available for use in 10 minutes, it will add ipv6 support for docker containers solving, can you validate it?

Will also release localdb and host.docker solvers in the next days.

Thanks in advance.

**Comment by 4ntsu** [comment]
Working perfectly here with the docker tag `3.15.2-snapshot`.
Muitissimo obrigado!
```
$ nslookup bookstack.docker
Server:		127.0.0.53
Address:	127.0.0.53#53

Non-authoritative answer:
Name:	bookstack.docker
Address: 192.168.50.101
Name:	bookstack.docker
Address: xxxx:xxxx:xxxx:xxxx::c
```

**Comment by mageddo** [comment]
> Working perfectly here with the docker tag `3.15.2-snapshot`. Muitissimo obrigado!
> 
> ```
> $ nslookup bookstack.docker
> Server:		127.0.0.53
> Address:	127.0.0.53#53
> 
> Non-authoritative answer:
> Name:	bookstack.docker
> Address: 192.168.50.101
> Name:	bookstack.docker
> Address: xxxx:xxxx:xxxx:xxxx::c
> ```

Tmj, man

**Comment by mageddo** [comment]
`3.15.5-snapshot` introduces `host.docker` IPV6 support

```
nslookup -po=8053 host.docker 127.0.0.1
Server:		127.0.0.1
Address:	127.0.0.1#8053

Non-authoritative answer:
Name:	host.docker
Address: 192.168.0.128
Name:	host.docker
Address: 2804:14x:xx:99a3:xxxx:b9b5:xxxx:ac36

```

**Comment by mageddo** [comment]
The complete feature was implemented since `3.15.7-snapshot`, see #389 for more details. Soon a final release.
